bin/crew

```diff
@@ -940,7 +999,7 @@ def expand_dependencies
 end
 
 def resolve_dependencies
-  abort "Package #{@pkg.name} is not compatible with your device architecture (#{ARCH}) :/".lightred unless @pkg.compatibility.include?('all') or @pkg.compatibility.include?(ARCH)
+  abort "Package #{@pkg.name} is not compatible with your device architecture (#{ARCH}) :/".lightred unless @device[:compatible_packages].any? do |elem| elem[:name] == pkgName end
```

Comment:
Faster? `unless @pkg.name.in? @device[:compatible_packages]`

Reply:
This might not work.  Just a thought.

Reply:
Definitely cleaner!